lib/rules/web/admin_console/4.8/dashboards.xyaml

```diff
@@ -667,6 +667,20 @@ check_charts_in_project_utilization:
   - selector:
       xpath: //h4[text()='Pod count']/ancestor::div[@data-test-id='utilization-item']//div[contains(@class,'pf-c-chart')]
     timeout: 40
+
+check_cpu_chart_orange_line:
```

Comment:
I would call it `check_cpu_request` instead of `check_cpu_chart_orange_line` because the color info can change at some time

Reply:
Done for change